Gemfile.lock

```diff
@@ -1,25 +1,131 @@
 GEM
   remote: https://rubygems.org/
   specs:
+    actionpack (8.0.2)
```

Comment:
All these gems are not going to be installed just because someone includes the gem, they're just added by rspec-rails as I've added tests for the ActionController extension.